lib/developer_portal/app/views/developer_portal/docs.html.liquid

```diff
@@ -1,21 +1,18 @@
-<h1>Documentation</h1>
-
-<p>Use our live documentation to learn about the Echo API</p>
+{% content_for javascripts %}
+    {{ 'active_docs.js' | javascript_include_tag }}
+{% endcontent_for %}
 
-{% cdn_asset /swagger-ui/2.2.10/swagger-ui.js %}
-{% cdn_asset /swagger-ui/2.2.10/swagger-ui.css %}
+<h1>Documentation</h1>
 
-{% include 'shared/swagger_ui' %}
```

Comment:
Now we don't use the partial `swagger_ui` anymore, we could delete https://github.com/3scale/porta/blob/master/lib/developer_portal/app/views/developer_portal/shared/_swagger_ui.html.liquid
Right @thomasmaas ?

Reply:
New liquid template of the documentation page follows https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.8/html/providing_apis_in_the_developer_portal/using-oas30#configuring_the_developer_portal_with_oas_3_0%23configuring_the_developer_portal_with_oas_3_0